### DISCUSSION TITLE
✨ Emby

### DISCUSSION THREAD

**Discussion by austinsr1** [runtipi/runtipi-appstore#3903] [Requests]
### Description

Emby Media Player

Emby is a versatile media server software that allows you to organize, stream, and play your personal media collection, including videos, music, and photos. With a user-friendly interface, Emby provides seamless access to your media across various devices such as smart TVs, smartphones, tablets, and computers. It offers features like live TV streaming, DVR, parental controls, and media sharing. Emby also supports metadata fetching, enabling detailed information and artwork for your media library. The platform emphasizes privacy, allowing you to control access and securely stream your content both locally and remotely. Emby is available on multiple platforms, including Windows, macOS, Linux, Android, and iOS.

### Official Website

https://emby.media/

### Documentation link

https://emby.media/support/articles/Home.html

### Last application release & date

4.8.8.0 (25/06/2024)

### Application license

commercial

### Source code repository link

closed source

### Docker image link

https://hub.docker.com/r/emby/embyserver

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by ABOhiccups** [comment]
Application license and Source code repository link are incorrect. Emby is open source not closed source and it's official GitHub repository exist.

Here's the correct information.

**Application license:**
GPL-2.0 license

**Source code repository link:**
[https://github.com/MediaBrowser/Emby](https://github.com/MediaBrowser/Emby)

**Comment by austinsr1** [comment]
Emby used to be opensource. An opensource fork exists called Jellyfin. The
repo you linked hasnt been updated in 6 years (version 3.5.2.0), when they went closed source.
The current license is available https://emby.media/license.html and the current version is 4.8.8.0 (25/06/2024)
